modules/utility/init.zsh

```diff
@@ -22,8 +22,12 @@ if [[ ! -z ${terminfo[colors]} ]] && (( ${terminfo[colors]} >= 8 )); then
     # BSD
 
     # colours for ls and completion
-    export LSCOLORS='exfxcxdxbxGxDxabagacad'
-    export LS_COLORS='di=34:ln=35:so=32:pi=33:ex=31:bd=36;01:cd=33;01:su=31;40;07:sg=36;40;07:tw=32;40;07:ow=33;40;07:'
+    if [[ -z ${LSCOLORS} ]]; then
```

Comment:
Hm... 76ff1bd was 6 months ago, 5.3 released Dec 11, 2016. Well, I personally am okay with only supporting the _latest_ version of zsh, as it makes our job easier. This means we may get complains from users about things not working, however in many cases means they're using some ancient mac-default version of zsh from 2011, the solution of "update your shit" is the same, so I'm actually okay with this.

`-z` it is.